api/src/main/java/org/pmiops/workbench/mail/MailServiceImpl.java

```diff
@@ -158,17 +145,11 @@ public void alertUserFreeTierExpiration(final DbUser user) throws MessagingExcep
     final String msgHtml =
         buildHtml(FREE_TIER_EXPIRATION_RESOURCE, freeTierExpirationSubstitutionMap(user));
 
-    final String subject = "Alert - Free credit expiration in All of Us Researcher Workbench";
-
-    final MandrillMessage msg =
-        new MandrillMessage()
-            .to(Collections.singletonList(validatedRecipient(user.getContactEmail())))
-            .html(msgHtml)
-            .subject(subject)
-            .fromEmail(workbenchConfig.mandrill.fromEmail);
-
     sendWithRetries(
-        msg, String.format("Free tier credits have expired for user %s", user.getUsername()));
+        user.getContactEmail(),
+        "Alert - Free credit expiration in All of Us Researcher Workbench",
+        String.format("Free tier credits have expired for user %s", user.getUsername()),
+        msgHtml);
```

Comment:
nit: change msgHtml to match expected arg name of htmlMessage like other calls to sendWithRetries